src/Form/FieldPassword.js

```diff
@@ -31,11 +38,14 @@ export default class FieldPassword extends FieldInput {
   getInputElement(attributes) {
     let classes = classNames(this.props.inputClass, this.props.sharedClass);
     attributes = this.bindEvents(attributes, this.props.handleEvent);
+    attributes.onBlur = this.handleOnBlur;
     attributes.onFocus = this.handleOnFocus;
 
-    let startValue = DEFAULT_PASSWORD_TEXT;
+    let fieldValue = attributes.defaultPasswordValue ||
+      attributes.startValue || '';
+
     if (this.focused) {
-      startValue = attributes.startValue;
+      fieldValue = attributes.startValue;
```

Comment:
I'm not sure, I didn't change this from the original implementation. But I can if you'd like.